src/main/resources/assets/paths/datasetIndex.yaml

```diff
@@ -0,0 +1,8 @@
+post:
+  summary: Index all datasets
+  description: Index all datasets in the system
+  tags:
+    - Dataset
```

Comment:
One request - can we add the `- Admin` tag here too so it shows up in the admin section? Same for index by id.